docs/manual/kr/buildTools/Testing-with-NPM.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This should be change to the actual language of the page:
```html
<html lang="kr">
```

Reply:
updated all <html lang="kr"> and doc/list.json files.